.github/workflows/build.yml

```diff
@@ -24,6 +24,19 @@ jobs:
     - name: build
       run: |
            python3 ci/build-in-all-configs.py --${{matrix.compiler}} --default-config
+  build-linux-bazel:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v4
+    - name: dependencies
+      run: |
+           sudo apt install libncurses5
```

Comment:
Why is ncurses needed?

Reply:
Clang16 needs this. But this is not a problem anymore because I upgraded the clang to 18 which does not need this lib.

Reply:
It looks like clang-18 still depends on this lib... so we sill need to install.